front/components/RenderMarkdown.tsx

```diff
@@ -161,6 +177,35 @@ export function RenderMarkdown({
   );
 }
 
+function MentionBlock({
+  agentName,
+  agentConfiguration,
+}: {
+  agentName: string;
+  agentConfiguration?: AgentConfigurationType;
+}) {
+  const statusText =
+    agentConfiguration?.status === "archived"
+      ? "[ARCHIVED] "
```

Comment:
We communicate about "deleting" an agent on the builder page, not archiving them so I would keep saying it's deleted. 

Suggestion: 
- If active : keep displaying the description. 
- If Archived:   "This assistant was deleted".
- If deactivated by the admin :  "This assistant was deactivated for this workspace". 
- if we can't find an agentConfiguration at all for this mention: "This assistant was deleted"  instead of "Description unavailable", because if we have this case it means that the agent was removed from db, which can only happen manually from a Dust eng 😅

Reply:
Archive / deleted : :+1: => [DELETED] rather than [ARCHIVED] in any case

Suggestion: yep, they're good messages :pray: Will add them in () after the description 

Although if you suggest not displaying the description:

I had discussed with  @Duncid who had a point that if the assistant is not available any more, but we're coming back to an old conversation, we still want the description of the agent to be able to understand what it did in the conversation => Therefore [STATUS] + we still see the description

There are edge cases where we really don't want the description anymore (i.e. leaked data in description) but they seem very edgy and can be handled in other ways (e.g. by changing the description before or after deleting), so for now keeping the description